src/main/java/Main.java

```diff
@@ -7,15 +7,23 @@ public class Main {
     public static void main(String[] args) throws IOException {
 
         File file = new File("employees.csv");
+        Employee[] employees = EmployeeDataReader.readFile(file);
+
+        CompanyStats calcStats = new CompanyStats();
 
         if (file.exists()) {
             File stats = new File("stats.txt");
             stats.createNewFile();
 
-//            FileWriter fileWriter = new FileWriter(stats);
-//            fileWriter.write("Średnia wypłata: 5000");
-//
-//            fileWriter.close();
+            FileWriter fileWriter = new FileWriter(stats);
+            fileWriter.write("Średnia wypłata: " + calcStats.getAvgSalary(employees) + "\n");
+            fileWriter.write("Minimalna wypłata: " + calcStats.getMinSalary(employees) + "\n");
+            fileWriter.write("Maksymalna wypłata: " + calcStats.getMaxSalary(employees) + "\n");
+            fileWriter.write(calcStats.getDepartmentEmployment(employees, "IT") + "\n");
+            fileWriter.write(calcStats.getDepartmentEmployment(employees, "Support") + "\n");
+            fileWriter.write(calcStats.getDepartmentEmployment(employees, "Management") + "\n");
+
+            fileWriter.close();
```

Comment:
Zapis do pliku wydziel do osobnej metody, podobnie jak zrobiłaś to z odczytem. Tu przydatne byłoby też stworzenie klasy, która przechowywała by dane ze statystykami, wtedy można by zapisać takie coś:
```
Employee[] employees = EmployeeDataReader.readFile(file);
Stats stats = calcStats.getStats)employees);
saveStats(stats, "company-stats.txt");
```